libnymea/interfaces/venetianblind.json

```diff
@@ -0,0 +1,13 @@
+{
+    "description": "Venetian blinds that can be tilted. Venetian blinds must support an angle and specify the minimum and maximum supported angle. For instance, if a venetian blinds supports tilting from horizontal to vertical by 90°, the minValue should be set to 0° and the maxValue to 90°. For venetian blinds that support tilting both direction, that is, a total of 180°, the minValue should be -90° and the maxValue should be 90°. 0° is always the horizontal position.",
+    "extends": "extendedblind",
+    "states": [
+        {
+            "name": "angle",
+            "type": "int",
+            "unit": "UnitDegree",
+            "minValue": "any",
+            "maxValue": "any"
```

Comment:
I fear we need to make this a device param so that it's user / api configurable per thing. In many cases motors are able to work with different kind of blinds. The manufacturer of the blind is usually a different from the one that provides the motors. It's not clear to me if we can auto-determine what exact blinds we're dealing with.

In case of Somfy 0/90 or -90/90 is a setting on the motor that we can read/set depending on the blind that is attached to the motor.

Reply:
Right now we decided to specify what the motor can do. If then the api reports that it can do only 0 - 90 or so, the plugin just needs to correct that... This is not perfect but also not exactly critical.

The proper solution would be to add support for requiring settingsTypes in interfaces, so the ui can take such settings into account... Or, allow dynamically setting min/max/allowed values as we've been talking before... Both would require somewhat bigger changes in the core/app tho, so let's go with this for now.